storage/innobase/log/log0recv.cc

```diff
@@ -762,14 +762,15 @@ void recv_sys_t::open_log_files_if_needed()
   }
 }
 
-void recv_sys_t::read(os_offset_t total_offset, span<byte> buf)
+MY_ATTRIBUTE((warn_unused_result))
+dberr_t
+recv_sys_t::read(os_offset_t total_offset, span<byte> buf)
```

Comment:
The return type should fit in the same line here. Only in the old InnoDB formatting style the return type was put in a separate line. The code is nicer to `grep` if the entire function signature (possibly excluding attributes) fits in a single line.